TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -309,7 +312,9 @@ private boolean validationSetCacheContainsSetWithName(String validationSetName)
   }
 
   private void doNewValidationSet() {
-    String name = JOptionPane.showInputDialog(MainWindow.getInstance(), "Enter new validation set name:");
+    String name = JOptionPane.showInputDialog(MainWindow.getInstance(), "Enter new validation set method name: " +
+            "\nThis is the name that will be used when generating the validation methods.\n" +
+            "\nPlease confirm that the input dialog looks correct with this additional text.");
```

Comment:
Is this line intentional?

Reply:
This is something you asked me to add. So is the context out of order here ?